### ISSUE TITLE
Add dual sim support on dual sim phones for effective stumbling.

### ISSUE THREAD

**Issue by bharat127** [mozilla/MozStumbler#1644]
It seems mozilla stumbler doesn't detect the cellid-info of second sim on dual sim phones.

Add dual sim support on dual sim phones, for a more effective stumbling.

**Comment by crankycoder** [comment]
Do you have a particular device that you use?  We currently have code in place to capture dual-sim but I don't think we've tested this on real hardware.  I can look into ordering a device to test with.

**Comment by bharat127** [comment]
yes, I do have the YU YUREKA smartphone with dual sim capability.
It is currently on custom built CyanogenMod 12 (android 5.0.2) corresponding to the stock stable CyanogenOS branch.

I can always help testing the feature.

**Comment by unaszplodrmann** [comment]
Stumbling on my Moto G (titan) only captures tower data from one SIM card. This is on CM 12.1... Have you guys looked at the API for multi-SIM added in Android 5.1?

**Comment by unaszplodrmann** [comment]
I found a way to get towers capture from both SIMs: With mobile data enabled on my 3 SIM in slot 1 using 3G, it was only capturing towers from that slot, but when I put my 3 SIM into the secondary slot and reverse the setup, so that 3G and mobile data are enabled for slot 2, all towers are captured. In both instances outgoing call selection is set to 'ask every time'.
